src/include/duckdb/common/unique_ptr_utils.hpp

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include "duckdb/common/winapi.hpp"
+
+namespace duckdb {
+
+struct __unique_ptr_utils {
+	DUCKDB_API static void AssertNotNull(void *ptr);
```

Comment:
```suggestion
	static void AssertNotNull(bool valid);
```
We are not using the actual pointer, and I don't think this is needed for API.

Reply:
(in case also the call sites can be changed to just passing in the this)